code/game/dna/dna_modifier.dm

```diff
@@ -589,6 +589,8 @@
 		var/lock_state = src.connected.locked
 		src.connected.locked = 1//lock it
 
+		nanomanager.update_uis(src) // show the splash while this works
```

Comment:
It's because of the 2 second delay on radiating a block, it refreshes after the 2 second delay, but not when the delay starts. So the splash that says "wait 2 seconds" barely shows up, if ever.
If performance is an issue, wouldn't it be better to REMOVE THE AUTOUPDATE?